mesheryctl/internal/cli/root/system/logs.go

```diff
@@ -41,8 +41,10 @@ import (
 var logsCmd = &cobra.Command{
 	Use:   "logs",
 	Short: "Print logs",
-	Long:  `Print history of Meshery's logs and begin tailing them.`,
-	Args:  cobra.ArbitraryArgs,
+	Long: `Print history of Meshery's logs and begin tailing them for the Kubernetes platform.
```

Comment:
This could just be "Print history of Meshery's container logs and begin tailing them."